src/common/marlin_client.c

```diff
@@ -569,6 +569,16 @@ void marlin_print_start(const char *filename) {
     _send_request_to_server_and_wait(request);
 }
 
+void marlin_set_current_file(const char *filename) {
```

Comment:
What happens here if this happens while printing?

Because it is possible (and desirable) to be able to upload while printing. Then the file is ready & waiting there once the job is finished. You can then just return to the home screen after cleaning the bed and it'll pop it up as the next one, avoiding several ways between the printer and the computer.

But I assume marlin already uses the "current file" while printing and changing it at that time is not the best course of action?